mpt/impl/imap-mailbox/core/src/main/resources/org/apache/james/imap/scripts/Annotation.test

```diff
@@ -0,0 +1,39 @@
+################################################################
+# Licensed to the Apache Software Foundation (ASF) under one   #
+# or more contributor license agreements.  See the NOTICE file #
+# distributed with this work for additional information        #
+# regarding copyright ownership.  The ASF licenses this file   #
+# to you under the Apache License, Version 2.0 (the            #
+# "License"); you may not use this file except in compliance   #
+# with the License.  You may obtain a copy of the License at   #
+#                                                              #
+#   http://www.apache.org/licenses/LICENSE-2.0                 #
+#                                                              #
+# Unless required by applicable law or agreed to in writing,   #
+# software distributed under the License is distributed on an  #
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       #
+# KIND, either express or implied.  See the License for the    #
+# specific language governing permissions and limitations      #
+# under the License.                                           #
+################################################################
+C: a0 CAPABILITY
+S: \* CAPABILITY .* ANNOTATION .*
+S: a0 OK CAPABILITY completed.
+
+C: a1 SETMETADATA INBOX (/private/comment "My new comment")
```

Comment:
We have a lot of duplication between UT and MPT but MPT is the most valuable part of our testsuite because we can make it run on any other mail server and check for behavior variations.
UT is here to help you code, MPT is here to stay for a long time, we need both.